frontend/src/helpers/object-to-query.ts

```diff
@@ -1,16 +1,52 @@
 export const objectToQueryString = (filters: {
-  [p: string]: string | string[];
+  [p: string]: string | string[] | Record<string, string | string[]>[];
 }): string[][] => {
   const notEmpties = Object.fromEntries(
     Object.entries(filters).filter(
-      ([_, value]) => value.length >= 1 && value !== '' && value[0] !== '',
+      ([_, value]) =>
+        value.length >= 1 &&
+        value !== '' &&
+        Array.isArray(value) &&
+        value[0] !== '',
     ),
   );
 
   return Object.entries(notEmpties).flatMap(([key, value]) => {
     if (typeof value === 'string') {
       return [[key, value]];
     }
-    return value.map((item) => [key, item]);
+    if (
+      Array.isArray(value) &&
+      (value as []).every((item) => typeof item === 'object')
+    ) {
+      return objectsArrayToQuery(
+        key,
+        value as {
+          [p: string]: string | string[];
+        }[],
```

Comment:
`as { [p: string]: string | string[]; }[]`

:eye::eye:

Reply:
Again, I don't understand why the Array.isArray(value) check doesn't work
Because TS doesn't understand what's really going on